### PR TITLE
[routing] Minor fixes

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

File: routing/geometry.cpp [hunk 1]
```diff
@@ -53,19 +53,11 @@ class RoadAttrsGetter
...
-        m_cityRoads.Load(cont.GetReader(CITY_ROADS_FILE_TAG));
+    if (cont.IsExist(CITY_ROADS_FILE_TAG))
+      m_cityRoads.Load(cont.GetReader(CITY_ROADS_FILE_TAG));
 
-      if (cont.IsExist(MAXSPEEDS_FILE_TAG))
-        m_maxSpeeds.Load(cont.GetReader(MAXSPEEDS_FILE_TAG));
-    }
-    catch (Reader::Exception const & e)
-    {
-      LOG(LERROR, ("File", cont.GetFileName(), "Error while reading", CITY_ROADS_FILE_TAG, "or",
```
Comment (biodranik) [inline routing/geometry.cpp:66]: Was this logged info useful for debugging? Now it's gone.

File: routing/index_graph_loader.cpp [hunk 2]
```diff
@@ -133,20 +133,28 @@ IndexGraphLoaderImpl::GraphPtrT IndexGraphLoaderImpl::CreateIndexGraph(NumMwmId
...
+    DeserializeIndexGraph(*value, m_vehicleType, *graph);
 
-  return graph;
+    LOG(LINFO, (ROUTING_FILE_TAG, "section for", value->GetCountryFileName(), "loaded in", timer.ElapsedSeconds(), "seconds"));
+    return graph;
+  }
+  catch (RootException const & ex)
+  {
+    LOG(LERROR, ("Error reading graph for", value->m_file));
+    throw;
```
Comment (biodranik) [inline routing/index_graph_loader.cpp:156]: What will happen in production here?

**Review by biodranik — APPROVED** [review]

File: routing/geometry.cpp (hunk 1 above)
Comment (vng) [reply routing/geometry.cpp:66]: Now logging is close to the caller side for better processing.

File: routing/index_graph_loader.cpp (hunk 2 above)
Comment (vng) [reply routing/index_graph_loader.cpp:156]: Same as before, it is caught up the stack with "Internal error".

**Pull request merged**